src/features/unified-share-modal/__tests__/SharedLinkSection.test.js

```diff
@@ -185,4 +185,17 @@ describe('features/unified-share-modal/SharedLinkSection', () => {
             expect(wrapper).toMatchSnapshot();
         });
     });
+
+    test('should render disabled create share link message', () => {
+        const sharedLink = { url: '', canChangeAccessLevel: true };
+        const item = { grantedPermissions: { itemShare: false } };
+
+        expect(getWrapper({ sharedLink, item })).toMatchSnapshot();
+    });
+
+    test('should render disabled remove share link message', () => {
+        const sharedLink = { url: 'https://example.com/shared-link', canChangeAccessLevel: false };
+
+        expect(getWrapper({ sharedLink })).toMatchSnapshot();
```

Comment:
Do we need to snapshot the whole wrapper?

Reply:
I agree; this component is a bit too big for snapshots. Maybe we could just check that the message in the tooltip is correct?

Reply:
👍